server.go

```diff
@@ -16,16 +18,11 @@ import (
 )
 
 func main() {
-	// Grab $PORT from env
-	port := os.Getenv("PORT")
-	if port == "" {
-		log.Fatal("Missing: PORT environment variable")
-	}
-
-	// Grab $DATABASE_URL from env
-	databaseURL := os.Getenv("DATABASE_URL")
-	if databaseURL == "" {
-		log.Fatal("Missing: DATABASE_URL environment variable")
+	var config common.Config
+	if err := envconfig.Process("hercules", &config); err != nil {
```

Comment:
Just to note that previous variable name PORT is now HERCULES_PORT. The same is for DATABASE_URL, it is now HERCULES_DATABASEURL (no underscore between DATABASE and URL). If this is desired, then lgtm :)